kratos/geometries/triangle_3d_3.h

```diff
@@ -421,8 +424,9 @@ template<class TPointType> class Triangle3D3
      * :TODO: could be replaced by something more suitable
      * (comment by janosch)
      */
-    virtual double DomainSize() const override {
-      return Area();
+    virtual double DomainSize() const override 
+    {
+        return Area();

```

Comment:
Maybe I'm missing something but why is the DomainSize() method returning the Area?